Kconfig.nrf

```diff
@@ -11,6 +11,4 @@ rsource "lib/Kconfig"
 rsource "drivers/Kconfig"
 rsource "ext/Kconfig"
 
-orsource "../nrfxlib/Kconfig.nrfxlib"
-
```

Comment:
I assume this is going to move nrfxlib out of the "Nordic nRF Connect" menu.

Assuming this is not desirable I wonder if there is anything we can do about that.

@ulfalizer : Is it possible to append Kconfig code into a prior-defined menu, as one can with options?

If not, would it be reasonable to add support for this?

Reply:
There are other options. We can include only the nrf repo as a zephyr module, and then keep the inclusion of `nrfxlib/CMakeLists.txt` and `nrfxlib/Kconfig.nrfxlib` in the nrf repository, effectively using a mix of zephyr modules and direct inclusion.
For that we would also need to invoke `west list` from here to "find" nrfxlib if we don't want to assume its location.

Reply:
The advantage of presenting nrfxlib as a zephyr module is that users can then easily use nrfxlib with upstream zephyr, simply by having it in their west manifest, which is a nice feature if you ask me.

Reply:
@SebastianBoe 
Nope, not possible. The menu structure is static, and being able to see all choice symbols regardless of where you "enter" a choice defined in multiple locations is a Kconfiglib-specific hack.

Stuff would get pretty complex if it didn't work like that, e.g. re. dependencies and implicit menus.

Try to avoid doing too much out-of-tree stuff. Always gets pretty messy from what I've seen.

Reply:
@SebastianBoe I think it's fine to move nrfxlib out of nRF Connect and have them side by side.

Reply:
@lemrey : Thank you for clarifying.